src/store/action.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from '@reduxjs/toolkit';
+
+export const changeCity = createAction('chahgeCity', (city: string) => ({payload: city}));
```

Comment:
Можно упростить. А ещё опечатка
```suggestion
export const changeCity = createAction<string>('changeCity');
```